api_spec.yaml

```diff
@@ -5950,6 +5950,167 @@ paths:
+                type: object
+                properties:
+                  status:
+                    $ref: '#/components/schemas/statusOK'
+                  users:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/bulkImportUserResponse'
+                  nextPaginationToken: 
+                    $ref: '#/components/schemas/paginationToken'
+        '400':
+          $ref: '#/components/responses/400'
+        
+        '401':
+          $ref: '#/components/responses/401'
+        
+        '404':
+          $ref: '#/components/responses/404'
+        
+        '500':
+          $ref: '#/components/responses/500'
+
+    post:
+      tags:
+        - Bulk Import
+      operationId: addBulkImportUsers
+      description: |
+        Add users for bulk import. Maximum 10000 users can be added in one request.
+      parameters:
+        - $ref: '#/components/parameters/api-key'
+        - $ref: '#/components/parameters/cdi-version'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                users:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/bulkImportUser'
+              required:
+                - users
+      responses:
+        '200':
+          description: Bulk Import Users added successfully
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    $ref: '#/components/schemas/statusOK'
+        '400':
+          description: One or more users have invalid schema
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: "Data has missing or invalid fields. Please check the users field for more details."
+                  users:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        index:
+                          type: number
+                          example: 0
+                          description: "Index of the user in the users array that has invalid schema"
+                        errors:
+                          type: array
+                          items:
+                            type: string
+                            description: "List of errors for the user"
+                            example: "Invalid recipeId for loginMethod. Pass one of emailpassword, thirdparty or, passwordless!"
+
+        '401':
+          $ref: '#/components/responses/401'
+
+        '404':
+          $ref: '#/components/responses/404'
+
+        '500':
+          $ref: '#/components/responses/500'
+
+    delete:
```

Comment:
swagger complains that delete API should not have a request body, you may need to pass this via query params

Reply:
Ended up changing the path to POST `bulk-import/users/remove`